src/index.ts

```diff
@@ -14,7 +14,15 @@
  * limitations under the License.
  */
 
-export { githubInsightsPlugin as plugin } from './plugin';
+export {
+  githubInsightsPlugin as plugin,
```

Comment:
Should we export the named plugin too ? i.e. githubInsightsPlugin 
That's the new pattern right ?

Reply:
Yup, good call. I'll modify this PR a bit to be more involved so we can conform to the new entity-less approach as well.